File: TiaCodegen/Commands/Convert.cs
using TiaCodegen.Interfaces;

namespace TiaCodegen.Commands
{
    public class Convert : BaseOperationOrSignal, IFunctionOperation
    {
        public Convert(params IOperationOrSignal[] operationOrSignals)
            : base(operationOrSignals)
        { }
    }
}

File: TiaCodegen-ts/src/Commands/Convert.ts
import { IOperationOrSignal } from '../Interfaces/IOperationOrSignal.js';
import { IFunctionOperation } from '../Interfaces/IFunctionOperation.js';
import { BaseOperationOrSignal } from './BaseOperationOrSignal.js';

export class Convert extends BaseOperationOrSignal implements IFunctionOperation {
    constructor(...operationOrSignals: IOperationOrSignal[]) {
        super(...operationOrSignals);
    }
}
